src/Requests/GenerativeModel/GenerateContentRequest.php

```diff
@@ -54,5 +55,11 @@ protected function defaultBody(): array
             ),
             'generationConfig' => $this->generationConfig?->toArray(),
         ];
+
+        if ($this->systemInstruction !== null) {
+            $body['system_instruction'] = $this->systemInstruction->toArray();
```

Comment:
https://ai.google.dev/api/generate-content#request-body
It looks like this parameter can be written in camelcase notation to be compatible with the other parameters.